plugins/catalog-react/src/hooks/useEntityOwnership.ts

```diff
@@ -79,16 +39,11 @@ export function useEntityOwnership(): {
   isOwnedEntity: (entity: Entity) => boolean;
 } {
   const identityApi = useApi(identityApiRef);
-  const catalogApi = useApi(catalogApiRef);
 
   // Trigger load only on mount
   const { loading, value: refs } = useAsync(async () => {
     const { ownershipEntityRefs } = await identityApi.getBackstageIdentity();
-    const catalogRefs = await loadCatalogOwnerRefs(
-      catalogApi,
-      ownershipEntityRefs,
-    );
-    return new Set([...ownershipEntityRefs, ...catalogRefs]);
+    return ownershipEntityRefs;
```

Comment:
Should we add the actual user ref too, in case the ownership refs don't contain it?

Reply:
Naaaaah, in that case the refs are incorrect imo. Besides, I think it's good to allow for a configuration where ownership isn't claimed through individual users